backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/descriptors/DescriptorUtils.kt

```diff
@@ -77,6 +78,14 @@ internal fun IrSimpleFunction.resolveFakeOverride(allowAbstract: Boolean = false
     return realSupers.first { allowAbstract || it.modality != Modality.ABSTRACT }
 }
 
+internal fun IrSimpleFunction.overrides(superFunctionSymbol: IrSimpleFunctionSymbol): Boolean {
+    if (superFunctionSymbol in overriddenSymbols) return true
+    return overriddenSymbols.any { it.owner.overrides(superFunctionSymbol) }
```

Comment:
The implementation is kinda exponential time in worst case.
I believe general utility method shouldn't be like this.
Its usage in `InteropLowering.kt` can be replaced by a simple name-based check, so we probably can avoid adding this method at all for now.